src/main/kotlin/racing/view/ConsoleRender.kt

```diff
@@ -3,21 +3,19 @@ package racing.view
 import racing.model.Car
 import racing.model.RoundHistory
 
-class ConsoleRender {
+object ConsoleRender {
 
-    companion object {
-        fun roundRendering(roundHistory: RoundHistory): String {
-            return roundHistory.result
-                .map { carRendering(it) }
-                .joinToString(separator = "\n")
-        }
+    fun roundRendering(roundHistory: RoundHistory): String {
+        return roundHistory.results
+            .map { carRendering(it) }
+            .joinToString(separator = "\n")
+    }
 
-        private fun carRendering(car: Car): String {
-            return String.format("%s : %s", car.name, distanceRendering(car))
-        }
+    private fun carRendering(car: Car): String {
+        return String.format("%s : %s", car.name, distanceRendering(car))
```

Comment:
Kotlin은 백틱을 지원하기 때문에 String format을 사용하지 않아도 됩니다! 
```suggestion
        return "${car.name} : ${distanceRendering(car)}"
```

Reply:
오 네 이부분은 저도 모르게 너무 당연시하게 쓰던 자바의 잔재이네요 코틀린스럽게 수정하겠습니다!